BParkingNano/python/nanoBPark_cff.py

```diff
@@ -23,7 +23,6 @@
 
 nanoSequence = cms.Sequence(nanoMetadata + 
                             vertexSequence +           
```

Comment:
I think that the removal of muonBParkSequence from here is causing the crash on MC, 
because nanoAOD_customizeMC is run before the nanoAOD_customizeMuonTriggerBPark
and muons are needed there (in the MC sequence) for muonBParkMC